docs/bn/docs/index.md

```diff
@@ -0,0 +1,464 @@
+<p align="center">
+  <a href="https://fastapi.tiangolo.com"><img src="https://fastapi.tiangolo.com/img/logo-margin/logo-teal.png" alt="FastAPI"></a>
+</p>
+<p align="center">
+    <em>FastAPI উচ্চ কার্যক্ষমতা সম্পন্ন, সহজে শেখার এবং দ্রুত কোড করে প্রডাকশনের জন্য ফ্রামওয়ার্ক।</em>
+</p>
+<p align="center">
+<a href="https://github.com/tiangolo/fastapi/actions?query=workflow%3ATest" target="_blank">
+    <img src="https://github.com/tiangolo/fastapi/workflows/Test/badge.svg" alt="Test">
+</a>
+<a href="https://codecov.io/gh/tiangolo/fastapi" target="_blank">
+    <img src="https://img.shields.io/codecov/c/github/tiangolo/fastapi?color=%2334D058" alt="Coverage">
+</a>
+<a href="https://pypi.org/project/fastapi" target="_blank">
+    <img src="https://img.shields.io/pypi/v/fastapi?color=%2334D058&label=pypi%20package" alt="Package version">
+</a>
+</p>
+
+---
+
+**নির্দেশিকা নথি**: <a href="https://fastapi.tiangolo.com" target="_blank">https://fastapi.tiangolo.com</a>
+
+**সোর্স কোড**: <a href="https://github.com/tiangolo/fastapi" target="_blank">https://github.com/tiangolo/fastapi</a>
+
+---
+
+FastAPI একটি আধুনিক, দ্রুততম এবং উচ্চ কার্যক্ষমতা সম্পন্ন, Python 3.6+ দিয়ে API তৈরির জন্য স্ট্যান্ডার্ড পাইথন টাইপ ইঙ্গিতের ভিত্তিতে ওয়েব ফ্রেমওয়ার্ক।
+
+এর মূল বৈশিষ্ট্য গুলো হলঃ
+
+- **গতি**: এটি **NodeJS** এবং **Go** এর মত সেরা-কার্যক্ষমতা সম্পন্ন (Starlette এবং Pydantic এর সাহায্যে)। [পাইথন দ্রুততম ফ্রেমওয়ার্ক গুলোর মধ্যে এটি একটি](#কার্যক্ষমতায়)।
+- **দ্রুত কোড করা**:বৈশিষ্ট্য বিকাশের গতি ২০০% থেকে ৩০০% বৃদ্ধি করে৷ \*
```

Comment:
> - **দ্রুত কোড করা**:বৈশিষ্ট্য বিকাশের গতি ২০০% থেকে ৩০০% বৃদ্ধি করে৷ \*

- **দ্রুত কোড করা**:বৈশিষ্ট্য তৈরির গতি ২০০% থেকে ৩০০% বৃদ্ধি করে৷ \*

Reply:
added it